packages/cli/lib/src/flutter_tools/flutter_helpers.dart

```diff
@@ -43,8 +44,13 @@ bool isGlobalVersion(String version) {
 
 String getFlutterSdkExec(String version) {
   // If version not provided find it within a project
+
   if (version == null || version.isEmpty) {
-    return whichSync('flutter');
+    final execPath = whichSync('flutter');
```

Comment:
Should we check if execPath is null or empty before telling the user that a version was found?

Reply:
Well, in previous PR I did check it in the command file itself. But the I didn't wanted to make too much changes to the way to structured the code. So now it as using flutter in path and if not found. it prints error 'flutter not found in path'.

I am cool with any way to like.